exegol/model/ExegolContainer.py

```diff
@@ -25,10 +25,20 @@ def __init__(self, docker_container: Container, model: Optional[ExegolContainerT
         self.__id: str = docker_container.id
         self.__xhost_applied = False
         if model is None:
+            image_name = ""
+            try:
+                # Try to find the attached docker image
+                docker_image = docker_container.image
+            except ImageNotFound:
+                # If it is not found, the user has probably forcibly deleted it manually
+                logger.warning(f"Some images were forcibly removed by docker when they were used by existing containers!")
+                logger.error(f"The '{docker_container.name}' containers might not work properly anymore and should also be deleted and recreated with a new image.")
+                docker_image = None
+                image_name = "[red bold]BROKEN[/red bold]"
```

Comment:
could this be changed to "missing image", for clarity's sake?

Reply:
For UI size reasons, we decided not to change.

Reply:
[internal discussion] keeping for better looking reasons, "missing image" would be too long